docs/principles/browser-support.md

```diff
@@ -0,0 +1,98 @@
+# Browser support principles
+
+We believe that our journalism should be accessible to everyone.
+
+## Supported browsers
+
+Most browsers will receive some sort of experience. A small minority don’t meet our security requirements
+such as supporting TLS (IE<6, Firefox 1). These browsers will not be able to see theguardian.com.
+
+## Recommended browsers
+
+These are [browsers that we publicly recommend](https://www.theguardian.com/help/recommended-browsers) our 
+audience use. They will receive the modern experience.
+
+If a browser version has more than 0.2% of traffic, it is considered to be a candidate for recommendation. 
+All browser versions above and including the minimum candidate version are considered recommended browsers.   
+
+As of 6th November 2018, our browser usage for browsers versions with over 0.2% of traffic over the last 14 days 
+looks like:
+
+Browser           | Min version | Usage >= version |
+----------------- | ----------- | ---------------- |
+Chrome            | 49          | 46.95%           |
+iOS Safari        | 10.2        | 26.97%           |
+Firefox           | 48          | 3.07%            |
+Safari            | 10          | 3.02%            |
+Internet Explorer | 11          | 2.79%            |
+Samsung Internet  | 7           | 2.01%            |
+Edge              | 17          | 1.31%            |
+UC Browser        | 12          | 0.30%            |
+Opera             | 56          | 0.25%            |
+
+The above table was calculated using [this spreadsheet](https://docs.google.com/spreadsheets/d/17g8py3P30nJLVqrlgc8JW4rkFhPTqNvwrMJdaOFnFzs/)
+based on data from Google Analytics.
+
+## Cutting the mustard
+
+_[Cutting the mustard](http://responsivenews.co.uk/post/18948466399/cutting-the-mustard)_ describes a technique that divides
+web browsers in to two groups - 'modern' and 'core'.
+
+We do this by evaluating the client's capability, like so:
+
+```
+var isModern = (function () { 
+  'use strict';
+  return Function.prototype.bind && !this;
+}());
+```
+
+The _modern_ experience is designed to work on everything, whereas the _core_ experience is progressively enhanced with JavaScript.
+
+For browsers that don't cut the mustard, we will deliver only the core experience. More modern browsers will receive the modern experience.
+
+This ensures a graceful experience when viewing with a less capable device; or when, for whatever reason, the JavaScript fails we can
+turn our [escalator into stairs](http://jakearchibald.com/2013/progressive-enhancement-still-important).
+
+### Core experience
+
+With the core experience, content is legible and styling is basic. 
```

Comment:
Yeah, I was thinking for older browsers we shouldn't make any special effort to write fallbacks for every new CSS language feature. I don't think it would be necessary to find a way of splitting up core and modern styles, from a browser's perspective.